lib/api_key.dart

```diff
@@ -0,0 +1,3 @@
+import 'dart:io' show Platform;
+
+String api_key = 'd95b5097-37d3-4c86-b320-4c42f66c129a';
```

Comment:
Please do not commit API Keys to the GitHub repo, add the file to gitignore

1. the import statement is unnecessary, why is it there?
2. Make a class and add a static field in the class, don't declare global variables like this
3. Don't use underscores, for constants, either use full uppercase or camelCase. looks cleaner that way